//package io.core.protocol
message DugeonItem
{
	optional int32	process		= 1;	//进度
	optional int32	remaintimes	= 2;	//剩余挑战次数
	optional int32	totaltimes	= 3;	//今日总次数
	repeated int32	stageids	= 4;	//式神碎片和ssr式神碎片当日的关卡ID
}

message DugeonData
{
	repeated DugeonItem		alldugeon	= 1;//所有地下城进度
	optional int32			buycost		= 2;//觉醒购买消耗勾玉数量
	optional int32			remainbuy	= 3;//剩余可购买次数
}

message DseDugeonOpt
{
	optional int32	res				= 1;	//返回码
	optional int32	type 			= 2;	//1请求数据地下城 2请求数据觉醒秘境
	optional DugeonData dugeondata 	= 3;	//地下城数据或者觉醒秘境数据
}

